### PR TITLE
Edit function

### PR DESCRIPTION
**Pull request by dorosiemers**
It's an editing function.

### CODE REVIEW

File: components/listEntry.js [hunk 1]
```diff
@@ -1,32 +1,94 @@
...
   handleRemove,
 }) {
+  const [isEditable, setIsEditable] = useState(false);
+  const [location, setLocation] = useState(data.location);
+  const [time, setTime] = useState(data.time);
+  const [offender, setOffender] = useState(data.offender);
+  const [incident, setIncident] = useState(data.incident);
+  function handleSave() {
+    setIncidents(
+      incidents.map((e) =>
```
Comment (mpagels) [inline components/listEntry.js]: Recommend using a more specific name for the variable, then just the `e`. `e` is often use for `event` in an `eventListener. ` This is definitely something other and `future-doro` will stumble. An easy solution would be to just using the singular version of the list name: `incident`.

File: components/listEntry.js [hunk 2]
```diff
@@ -1,32 +1,94 @@
...
+        e.id === data.id
+          ? { id: data.id, location, time, offender, incident }
+          : e
+      )
+    );
+    setIsEditable(false);
+  }
```
Comment (mpagels) [inline components/listEntry.js]: Is this whole function not an `update` rather than just a `save` ? just asking.

**Review by mpagels — APPROVED** [review]
Hey Doro, just found some minor improvements. Good progress here.

File: components/listEntry.js (hunk 2 above)
Comment (dorosiemers) [reply components/listEntry.js]: It saves the update.😅

**Pull request merged**